### DISCUSSION TITLE
how to limit to show only allow page

### DISCUSSION THREAD

**Discussion by ayemin-tun** [filamentphp/filament#8772] [Help]
### Description 

Hello I am new to filament and i use this path to show all of the resources 

`->discoverResources(in: app_path('Filament/Resources'), for: 'App\\Filament\\Resources') `

But I only want to show allow page like only Employee and Category page how to filter with this pls.

**Comment by leandrocfe** [comment]
Not sure if I understood the question, but you can use [policies](https://filamentphp.com/docs/3.x/panels/resources/creating-records#authorization) to show/hide resource pages
